### PR TITLE
Add —ignore-invalid-pages option

### PR DESCRIPTION
**Pull request by josiasmontag**
See #44 

Adds:
`pdfion-cli render --pages '1-10' --ignore-invalid-pages`

### CODE REVIEW

File: pdf/pdf.go [hunk 1]
```diff
@@ -48,21 +48,35 @@ func NormalizePageRange(pageCount int, pageRange string) (*string, *int, error)
...
 				parsedPageNumber, err := strconv.Atoi(pageRangeParts[pageRangePartI])
 				if err != nil {
+					if ignoreInvalidPages {
+						continue
+					}
 					return nil, nil, fmt.Errorf("%s is not a valid page number", pageRangeParts[pageRangePartI])
 				}
 
 				if parsedPageNumber < 1 || parsedPageNumber > pageCount {
+					if ignoreInvalidPages {
```
Comment (jerbob92) [inline pdf/pdf.go:76]: Why add the last page here?

**Review by jerbob92 — COMMENTED** [review]
I have merged and released as v0.9.0, however, during second review I found some issues, could you take a look?

File: pdf/pdf.go (hunk 1 above)
Comment (josiasmontag) [reply pdf/pdf.go:76]: If you use page range `1-10` but the file has only 5 pages, this corrects the range to `1-5`.

File: pdf/pdf.go [hunk 2]
```diff
@@ -48,21 +48,35 @@ func NormalizePageRange(pageCount int, pageRange string) (*string, *int, error)
 			} else if strings.HasPrefix(pageRangeParts[pageRangePartI], "r") {
 				parsedPageNumber, err := strconv.Atoi(strings.TrimPrefix(pageRangeParts[pageRangePartI], "r"))
 				if err != nil {
+					if ignoreInvalidPages {
```
[the remark this answers is not in the document]
Comment (josiasmontag) [reply pdf/pdf.go:51]: Yea you could also just throw an error here.

File: pdf/pdf.go [hunk 3]
```diff
@@ -48,21 +48,35 @@ func NormalizePageRange(pageCount int, pageRange string) (*string, *int, error)
...
+						continue
+					}
 					return nil, nil, fmt.Errorf("%d is not a valid page number, the document has %d page(s)", pageCount-parsedPageNumber, pageCount)
 				}
 
 				pageNumbers = append(pageNumbers, pageCount-parsedPageNumber)
 			} else {
 				parsedPageNumber, err := strconv.Atoi(pageRangeParts[pageRangePartI])
 				if err != nil {
+					if ignoreInvalidPages {
```
[the remark this answers is not in the document]
Comment (josiasmontag) [reply pdf/pdf.go:69]: Yea you could also just throw an error here.

**Pull request merged**